### DISCUSSION TITLE
HMS-2000 with 48V Battery - Power per channel varies by a few watts

### DISCUSSION THREAD

**Discussion by AndreasBoehm** [hoylabs/OpenDTU-OnBattery#1182] [Q&A] [answered]
I am using a HMS-2000 connected to a 48V Battery from Pytes.
All cables (6mm2) should have roughly the same length.
I used three Phoenix Contact Reihenklemmen that are bridged together using Steckbrücken as the DC bus for the inverter alone.

I connected it as described here. Each column is one Reihenklemme.
| 1 | 2 | 3 |
|--|--|--|
| ch1 | Input from Busbar (16mm2)| ch2 |
| Bridged with 2 | Bridged with 1 | |
|  | Bridged with 3 | Bridged with 2 |
| ch3 |  | ch4 |

![connection](https://github.com/user-attachments/assets/23abbcd5-0c5d-44a1-a3a7-614e0ce68992)

I am still getting a difference of a couple of Watts per channel, even though the voltage is almost the same for all channels.
![Screenshot 2024-08-21 at 12 01 14](https://github.com/user-attachments/assets/70d1a90b-2f96-486c-8943-ffba6db37f82)

Any hints?

EDIT: The same happens with my HM-600 that is connected in a similar fashion.
![Screenshot 2024-08-21 at 12 11 06](https://github.com/user-attachments/assets/5013a5ec-7cb8-4351-a436-77d7bdbeb690)

**Comment by rainer667** [comment]
Hello,
I think that's normal because every MPPT tries to achieve the best value for itself. I had much worse values ​​because I had different cable lengths. This only got better after I shortened the cables to the same length. 
Sorry for my English, Google Translate

Hallo,
ich denke das ist normal, weil jeder MPPT versucht für sich den besten Wert zu erziehlen. Ich hatte viel schlimmere Werte weil ich verschiedene Kabellängen hatte. Das wurde erst besser nach dem ich Kabel auf die gleiche Länge gekürzt hatte.
Sorry für mein Englisch, Google Übersetzer

**Comment by AndreasBoehm** [comment]
Danke Rainer. Ich dachte mir schon das es einfach an den MPPTs liegt und eventuell noch am sehr niedrigen Limit und das der WR in dem Bereich einfach nicht sauber arbeiten kann.

**Comment by schlimmchen** [comment, marked as answer]
Meiner Meinung nach ist das ganz normale Streuung, die man kaum wegoptimieren kann. Hier meiner:
![image](https://github.com/user-attachments/assets/9450cd4d-e9d2-40c4-a538-00207296b48e)

Der meines Vaters hat eine deutlich schieflast, weil ich es scheinbar nur einmal gesagt habe, dass die Kabel gleich lang werden müssen, was offensichtlich nicht ausreichte um diese Anforderung in seinem Kopf zu manifestiren :wink: Die Kabel sind unterschiedlich lang geworden aufgrund der räumlichen Anordnung des Verteilerkastens und des WR. Kanal 3 und 4 produzieren ständig signifikant weniger und die Gesamtleistung (AC) liegt bei nur ~1890W dadurch.
![image](https://github.com/user-attachments/assets/dd558dd7-3c88-41f6-9a78-c779e6a525f3)

**Reply by AndreasBoehm** [reply]
Vielen Dank für die Screenshots, auch Danke an deinen Vater das er uns hier ganz deutlich aufzeigt wie wichtig es ist das die Kabel gleich lang sein sollten 😆 

Dachte mir schon fast das diese Abweichungen normal sind, dann hab ich jetzt die Bestätigung dafür bekommen.

**Reply by spcqike** [reply]
> ~1890W

das ist  .... interresant und erschreckend. Hast du die Möglichkeit das genauer zu untersuchen? 100W Verlust sind nicht ohne.

Wobei ich nicht ganz nachvollziehen kann, wie der Verlust aufgrund längerer Leitungen zustande kommt. Ein höherer Widerstand, klar. aber der Widerstand bewirkt eigentlich einen Spannungsabfall. Multipliziert mit dem fließenden Strom hast du deinen Leistungsverlust, der am Kabel oder den Klemmstellen als Wärme verschwindet.
Nur, hier sind die Spannungen gleich. der HMS zieht einfach weniger Strom.

ah, wo ich das schreibe fällt es mir glaube wieder ein... 
das Problem ist die geteilte Masse, DC(-), und die Shuntmessung im DC(-) eines jeden Eingangs, stimmts? 
Strom, der auf S3 und S4 reinkommt, fließt über S1 und S2 zurück, da die Kabel kürzer sind. Die Leistung wird in Wirklichkeit auf S3 und S4 umgesetzt, aber teilweise auf S1 und S2 gezählt. Und da S1 und S2 bereits "Fremdströme" zählen, setzten die MPPT dort entsprechend weniger Leistung um, als es den Anschein hat, da sich der MPPT auf ~10A drosselt. blöd nur, wenn bereits 2A da sind, die es eigentlich nicht gibt :D 

na immerhin keine Verlustleistung und keine Abwärme oder schlimmeres.

**Reply by ranma** [reply]
> das Problem ist die geteilte Masse, DC(-), und die Shuntmessung im DC(-) eines jeden Eingangs, stimmts?

Genau. Im idealfall müsste man sich mit MOSFETs einen aktiven Stromspiegel bauen der evtl. geringfügige Unterschiede in den Zuleitungswiderständen ausgleicht, aber da hat man dann natürlich auch nur extra Verlustleistung durch.

Letztlich ist das bei 24V relevanter als bei 48V, da bei 48V genug Reserve da ist. Bei 24V und starker Schieflast könnte der WR zu früh den Strom auf einem der MPPT-Eingänge begrenzen.

**Comment by ranma** [comment]
Mit vorkonfektioniertem Y-Kabel direkt am HM-800 ist der Unterschied minimal (ist dann nur ein 6qmm Adernpaar zum Verteiler, aber das ist auch kurz):
![image](https://github.com/user-attachments/assets/9ccea1c9-77a7-4bdf-a99a-9d61c1a7f670)

**Comment by tonibbb** [comment]
Bei mir ist es leider so, dass die Kanäle der bzw. die neuen WR nach einer gewissen Zeit immer stärker auseinanderdriften (24V-System). Sprich direkt nach Installation sind Spannung und Strom beider Kanäle gleich. Im Zeitraum einiger Monate vergrößert sich der Drift. Aktuell liegt der Unterschied bei etwa 30% über den gesamten Leistungsbereich. Und ja, das führt zu starken Abweichungen bei gewünschter Nulleinspeisung und leider dann auch zu häufigem Abschalten des/der WR. Vermute irgendwelche Alterungserscheinungen im WR aufgrund unterschiedlicher Leistungen der Kanäle bei geteilter Masse (?).

**Reply by spcqike** [reply]
Ich kann deiner Beschreibung nicht folgen. 

> Vermute irgendwelche Alterungserscheinungen im WR aufgrund unterschiedlicher Leistungen der Kanäle bei geteilter Masse (?).

das einzige was ich mir in dem Zusammenhang vorstellen kann ist eine Beschädigung einzelner Eingänge durch unbeabsichtigte Überlastung. Das ist aber keine Alterung im herkömmlichen Sinne. 

Hast du eine Batterie oder mehrere? Was meinst du mit 30% drift? Sind alle Kabel wirklich fest oder ist was lose? 

Wenn es am Anfang gleichmäßig verteilt war, sollte das eigentlich nicht verschwinden. Außer es löst sich etwas oder der WR geht kaputt.

**Reply by tonibbb** [reply]
Die WR und Eingänge sind seit Installation unverändert. Da wackelt auch nichts bzw. keiner dran rum. Die angezeigten Spannungen der Eingänge sind weitestgehend konstant gleich (ja, manchmal sind hier Unterschiede von bis zu 0,2 V zu sehen). Anfangs waren auch die angezeigten Ströme identisch, jedoch weicht einer der Kanäle hier über den Zeitraum von mehreren Monaten immer stärker ab. Aktuell sind das etwa 30%. Sprich z.B. bei gleicher Spannung von 25 V hat Kanal 1 z.B. 100 W und der andere nur 60 - 70. Dieses Verhalten habe ich bisher bei mehreren WR, sowohl HM600 als auch HMS800 beobachtet. Da sich außen nichts gelöst hat gehe ich eben auch davon aus, dass die WR langsam kaputt gehen. Interessanterweise funktioniert die Nulleinspeisung zu manchen Tageszeiten sehr gut, zu anderen Tageszeiten eher gar nicht und der WR oder die DTU schaltet ab.

**Reply by ranma** [reply]
> Bei mir ist es leider so, dass die Kanäle der bzw. die neuen WR nach einer gewissen Zeit immer stärker auseinanderdriften (24V-System). Sprich direkt nach Installation sind Spannung und Strom beider Kanäle gleich. Im Zeitraum einiger Monate vergrößert sich der Drift. Aktuell liegt der Unterschied bei etwa 30% über den gesamten Leistungsbereich. Und ja, das führt zu starken Abweichungen bei gewünschter Nulleinspeisung und leider dann auch zu häufigem Abschalten des/der WR. Vermute irgendwelche Alterungserscheinungen im WR aufgrund unterschiedlicher Leistungen der Kanäle bei geteilter Masse (?).

Wie genau sind die Eingänge verbunden? Vieleicht irgendwo eine Klemme nicht fest genug angezogen oder schlecht gecrimpt und durch Temperaturzyklen erhöht sich nach und nach der Übergangswiderstand?

**Comment by AndreasBoehm** [comment]
Hab mir diesmal extra viel Mühe gegeben beim Kabel auf Länge schneiden und beim konfektionieren der Stecker. Ist jetzt aber auch schon das vierte System.

Gleiche Spannung an allen vier Eingängen 🎉 

![Screenshot 2024-09-09 at 13 47 17](https://github.com/user-attachments/assets/51ce0805-7926-4040-9517-300c00bb4369)

**Reply by schlimmchen** [reply]
Nice :rocket: 

Ist das dein HMS-2000? Mach den Screenshot nochmal wenn dessen Limit auf 2000W steht :grin:

**Reply by AndreasBoehm** [reply]
Jap, ist mein zweiter. Hier ein Screenshot mit 2000W Limit :)
![HMS-2000](https://github.com/user-attachments/assets/61522db6-6f31-46d3-83cc-46ed022327a0)

**Reply by spcqike** [reply]
das kannst du so aber nicht stehen lassen, da fehlen dir ja 9W ;) 
![grafik](https://github.com/user-attachments/assets/d3c5405c-8e14-4c7e-b059-134d7289f4c2)

(ohne Batterie, also auch ohne Masseprobleme)

**Reply by schlimmchen** [reply]
Der macht uns was vor, Andreas...
![Screenshot from 2024-09-10 12-48-29](https://github.com/user-attachments/assets/e2007b55-a263-4f2c-96fa-0f5e068e4d74)
Warum ist dessen HMS-2000 besser ausbalanciert? Ich hab echt peinlich genau auf gleich lange Leitungen und gute Crimps geachtet. Ich denke ich zieh mal die Schrauben am Klemmblock nach^^

**Reply by spcqike** [reply]
> Der macht uns was vor, Andreas...
Warum ist dessen HMS-2000 besser ausbalanciert?

wie gesagt, ohne Batterie. rein PV (2x Trina Vertex S+ 435W parallel pro Eingang). da gibts kein Problem mit der geteilten Masse zwischen den Eingängen :) 
Mein Kommentar war natürlich auch scherzhaft gemeint und war als Referenz des maximal Möglichen gedacht :) (Fertigungstoleranzen und Temperatureinflüsse mal außen vor, hätte er keine 52°C wären vielleicht auch 1-2W mehr drin :) )

**Reply by schlimmchen** [reply]
Achsoooo, das hab ich total vergessen...

Moment mal... Was hast denn du bitte für Panele, die 540W ausspucken?! Um diese Zeit? Wird jetzt offtopic, aber ich wills wissen^^

**Reply by spcqike** [reply]
> Was hast denn du bitte für Panele, die 540W ausspucken?

> 2x Trina Vertex S+ 435W parallel pro Eingang

8 Trina, flach auf dem Garagen Dach, Süd/ost, ca 5 grad Neigung. Aber bis späten Nachmittag größtenteils ohne Schatten. 
🤷‍♂️
